packages/amesos2/src/Amesos2_Basker_decl.hpp

```diff
@@ -99,7 +99,7 @@ class Basker : public SolverCore<Amesos2::Basker, Matrix, Vector>
   // TODO: Would like to change dtype to be a regular type, not static.
```

Comment:
I think this comment can now be removed

Reply:
@ndellingwood Thanks I removed this - if you can please review again.